### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by MishterKirby**
Fixed typos

### CODE REVIEW

**Comment by timg236** [comment]
Thanks for the PR - you'll need to move it out of draft before we can merge it

**Comment by MishterKirby** [comment]
Ok, let me double-check it, and I'll take it out of draft.

**Comment by MishterKirby** [comment]
Ok, done.

File: firmware/release-notes.md [hunk 1]
```diff
@@ -70,32 +70,32 @@ https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711_bootloade
 
 ## 2020-09-03 Only use green LED for error status in bootloader - BETA
    * Turn the green LED on and leave it on unless an error code occurs.
-     Previously, SD activity was displayed but that plus muxing with the
+     Previously, SD activity was displayed but that plus mixing with the
```
Comment (lurch) [inline firmware/release-notes.md:73]: @timg236 I think this should have been left as `muxing` ?

File: firmware/release-notes.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711_bootloade
      issue which does not impact Model B
    * Update the VL805 embedded / standalone FW version to 0138A1
       *  User settings of the ASPM bits in the PCI configuration space
-         link control register are now maintained
+         link control registers are now maintained
```
Comment (lurch) [inline firmware/release-notes.md:112]: I've got no idea if the change here from "register" to "registers" was correct?

File: firmware/release-notes.md (hunk 1 above)
Comment (MishterKirby) [reply firmware/release-notes.md:73]: Whoops! I should've told Word to ignore those. Welp

File: firmware/release-notes.md (hunk 2 above)
Comment (MishterKirby) [reply firmware/release-notes.md:112]: This one too, same as above

**Pull request merged**